docker-compose/technical/grafana/config/datasources/datasource.yml

```diff
@@ -0,0 +1,45 @@
+apiVersion: 1
+
+datasources:
+    - name: Prometheus
+      type: prometheus
+      access: proxy
+      url: http://prometheus:9090
+      editable: false
+      jsonData:
+        httpMethod: POST
+        exemplarTraceIdDestinations:
+          - name: trace_id
+            datasourceUid: tempo
+    - name: Tempo
+      type: tempo
+      access: proxy
+      orgId: 1
+      url: http://tempo:3200
+      basicAuth: false
+      isDefault: true
+      version: 1
+      editable: false
+      apiVersion: 1
+      uid: tempo
+      jsonData:
+          httpMethod: GET
+          tracesToLogs:
+              datasourceUid: 'loki'
+    - name: Loki
+      type: loki
+      uid: loki
+      access: proxy
+      orgId: 1
+      url: http://loki:3100
+      basicAuth: false
+      isDefault: false
+      version: 1
+      editable: false
+      apiVersion: 1
+      jsonData:
+          derivedFields:
+              -   datasourceUid: tempo
+                  matcherRegex: \[.+,(.+?),
+                  name: TraceID
+                  url: $${__value.raw}
```

Comment:
This can be deleted as it links to Tempo and Loki instances (not deployed here)

Reply:
We are going to use them